pages/design-principles/design-principles.md

```diff
@@ -242,7 +242,7 @@ The following are practical actions you can take:
 - [Example: USDA's design and brand guidelines](https://www.usda.gov/digital-strategy/design-and-brand) [usda.gov]
 - [Example: Centers for Medicare and Medicaid Services design system](https://design.cms.gov/) [design.cms.gov]
 - [Federal Front Door research findings](https://labs.usa.gov/) [labs.usa.gov]
-- [Sharing Quality Services Cross-Agency Performance goal](https://www.performance.gov/CAP/sharing-quality-services/) [performance.gov]
+- [Sharing Quality Services Cross-Agency Performance goal](https://trumpadministration.archives.performance.gov/CAP/sharing-quality-services/) [performance.gov]
```

Comment:
I believe I found this content, but since it is archived I wonder if we should remove.

Reply:
Makes sense to me, do you agree @thisisdano?

Also, what's with the extra space after links? For example:
```md
[Federal Front Door research findings](https://labs.usa.gov/) [labs.usa.gov]
```

Reply:
Forgive me if I misunderstood your question, but looks like it is just letting users know the link source.

Reply:
@amyleadem no, that makes sense. Thank you!